Update unarchive.py - Further clarify extra_opts

jagibson opened the pull request:
##### SUMMARY
Update the description in the options to help assist with what the module is looking for in the extra_opts.  #31873 alone is not enough since if you are not familiar with the option given in the example it may not be obvious that both elements are part of the same option instead of being two different options.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Further clarifies #31873.

Also the Ansible output is not clear what the error is if you format the extra_opts incorrectly (see below)
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
unarchive.py module

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below
Failed to find handler for \"/var/tmp/blah.tar.gz\". Make sure the required command to extract the file is installed. Command \"/bin/gtar\" could not handle archive. Command \"/usr/bin/unzip\" could not handle archive.
```

jagibson committed:
Update unarchive.py - Further clarify extra_opts

##### SUMMARY
Update the description in the options to help assist with what the module is looking for in the extra_opts.  #31873 alone is not enough since if you are not familiar with the option given in the example it may not be obvious that both elements are part of the same option instead of being two different options.


##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

jagibson committed:
Fix for line being too long

jagibson commented:
This is what I get for editing the docs in github directly.  I'll have to try this locally.

acozine commented on lib/ansible/modules/files/unarchive.py:
```diff
@@ -71,7 +71,8 @@
   extra_opts:
     description:
-      - Specify additional options by passing in an array.
+      - Specify additional options by passing in an array.  Each space-separated part of the archive program's command-line option should be a new element\
```
```suggestion
      - Specify additional options by passing in an array.
      - Each space-separated command-line option should be a new element of the array. See examples.
```

acozine commented on lib/ansible/modules/files/unarchive.py:
```diff
@@ -71,7 +71,8 @@
     description:
-      - Specify additional options by passing in an array.
+      - Specify additional options by passing in an array.  Each space-separated part of the archive program's command-line option should be a new element\
+      of the array.
```
```suggestion
    - Command-line options with multiple elements must use multiple lines in the array, one for each element. 
```

jagibson replied:
Sorry this got lost - I blame github for hiding old notifications :).

I've accepted the changes.

acozine reviewed (commented):
@jagibson, I think accepting these two suggestions will get CI passing.

acozine commented:
@jagibson re-read the PR and example - I misunderstood what you were saying at first. Do the updated suggestions express your meaning correctly?

jagibson committed:
Update lib/ansible/modules/files/unarchive.py

Co-Authored-By: Alicia Cozine <879121+acozine@users.noreply.github.com>

jagibson committed:
Update lib/ansible/modules/files/unarchive.py

Fix for multi-line

Co-Authored-By: Alicia Cozine <879121+acozine@users.noreply.github.com>

jagibson committed:
Fix indent

jagibson committed:
Link checker doesn't like "yes"?

ERROR: lib/ansible/modules/files/unarchive.py:0:0: doc-default-does-not-match-spec: Argument 'validate_certs' in argument_spec defines default as (True) but documentation defines default as (False) (75%)

Changed doc to be "True" which matches the code.

jagibson committed:
Delete trailing whitespace

jagibson committed:
change validate_certs default back to "yes"

Maybe it was a fluke that it failed before.

acozine commented:
This looks good, thanks @jagibson.
